conda_smithy/anaconda_token_rotation.py

```diff
@@ -32,17 +36,17 @@ def _get_anaconda_token():
 
 
 def rotate_anaconda_token(
-    user,
-    project,
-    feedstock_config_path,
-    drone=True,
-    circle=True,
-    travis=True,
-    azure=True,
-    appveyor=True,
-    github_actions=True,
-    token_name="BINSTAR_TOKEN",
-    drone_endpoints=(),
+    user: str,
+    project: str,
+    feedstock_config_path: Optional[str],
+    drone: bool = True,
+    circle: bool = True,
+    travis: bool = True,
+    azure: bool = True,
+    appveyor: bool = True,
+    github_actions: bool = True,
+    token_name: str = "BINSTAR_TOKEN",
+    drone_endpoints = (),
```

Comment:
*Question*: this is a type that doesn't exist? can't we at least use `list` or function? (what is this?)

Reply:
it seems like `list | tuple` worked. Only adding `list` was throwing error that `()` is not a list

Reply:
MY best guess is that `()` is the empty tuple, which is not a very useful value to pass to any function.